### PR TITLE
Added support for SameSite cookies.

### PR DESCRIPTION
**Pull request by akash0x53**
Good to mitigate against CSRF attacks. `SameSite` cookies are mentioned
in draft https://tools.ietf.org/html/draft-west-first-party-cookies-07

Fix for issue #410

### CODE REVIEW

**Comment by akash0x53** [comment]
@anandology Please review & comment.

**Comment by anandology** [comment]
@akash0x53 Did you notice my comment about string concatenation?

**Comment by akash0x53** [comment]
Nope. This is comment on PR :-)

File: web/webapi.py [hunk 1]
```diff
@@ -389,9 +389,12 @@ def setcookie(name, value, expires='', domain=None,
         morsel['domain'] = domain
     if secure:
         morsel['secure'] = secure
-    value = morsel.OutputString()
     if httponly:
-        value += '; httponly'
+        morsel["httponly"] = True
+    value = morsel.OutputString()
+    if samesite and samesite.lower() in ['strict', 'lax']:
+        samesite_attr = "; SameSite=%s" % samesite
```
Comment (akash0x53) [inline web/webapi.py]: Good to go? @anandology

**Comment by akash0x53** [comment]
SameSite support PR open for Django too. 
https://github.com/django/django/pull/8380/files

**Comment by akash0x53** [comment]
Any update on this PR?

**Comment by jzellman** [comment]
:+1: Looks good to me, @akash0x53 maybe squash down to a single commit?

**Pull request closed without merging**